src/applications/vaos/referral-appointments/components/ReferralLayout.jsx

```diff
@@ -41,8 +42,20 @@ function BreadCrumbNav() {
   );
 }
 
-export default function ReferralLayout({ children, hasEyebrow }) {
+export default function ReferralLayout({
+  children,
+  hasEyebrow,
+  apiFailure,
+  heading,
+}) {
   const location = useLocation();
+  let headingTestId = null;
+  if (heading) {
+    headingTestId = `${heading
```

Comment:
this could be a useful util maybe

Reply:
I removed it instead. I can test for the heading text in ReferralLayout unit test and everywhere else I changed it to look for "referral-layout-heading"